templates/zerver/api/roles-and-permissions.md

```diff
@@ -0,0 +1,85 @@
+# Roles and permissions
+
+Zulip offers [several levels of permissions based on user
+organization-level roles](/help/roles-and-permissions).
+
+Here are some important details to note when working with these
+roles and permissions in Zulip's API:
+
+## A user's organization-level role
+
+As of [Zulip 4.0](/api/changelog#changes-in-zulip-40) (feature levels
+59 and 60), endpoints and [events](/api/get-events#realm_user-add)
+that return user account data include a `role` property, which
+contains the user's organization-level role. These roles are encoded as:
+
+* Organization owner: 100
+
+* Organization administrator: 200
+
+* Organization moderator: 300
+
+* Member: 400
+
+* Guest: 600
+
+User account data also include these relevant boolean properties:
+
+* `is_admin` specifying whether the user is an organization administrator.
+
+* `is_guest` specifying whether the user is a guest user.
+
+* `is_owner` specifying whether the user is an organization owner, and
+  present as of [Zulip 3.0](/api/changelog#changes-in-zulip-30) (feature
+  levels 8 and 11). If true, `is_admin` will also be true.
+
+* `is_billing_admin` specifying whether the user is a billing administrator,
+  and present as of [Zulip 5.0](/api/changelog#changes-in-zulip-50)
+  (feature level 73).
+
+Additionally, as of [Zulip 4.0](/api/changelog#changes-in-zulip-40)
+(feature level 60), when `realm_user` is present in `fetch_event_types`,
+the [`/register`endpoint](/api/register-queue) will return an
+`is_moderator` boolean value specifying whether the user is an organization
+moderator.
+
+Note that the [`GET users/me`](/api/get-own-user) endpoint did not
+return a `is_guest` value until
+[Zulip 3.0](/api/changelog#changes-in-zulip-30) (feature level 10).
+
+## Permissions levels
+
+Many areas of Zulip are customizable by the organization-level roles
+above, such as (but not limited to) [configuring message editing and
+deletion](/help/configure-message-editing-and-deletion) and
+[streams permissions](/help/stream-permissions).
+
+These permission levels and policies in the API are designed to be
+cutoffs in that users with the specified role and above have the
+specified ability or access.
+
+The most trusted, and therefore most restrictive, permission level or policy
+would be organization owner only, or nobody. And the least trusted, and
+therefore least restrictive, permission level would be members only,
```

Comment:
This makes it sound like "members only" == "everyone", which is not true. It might be simpler to actually just list the possiblities:
* Everyone (least restrictive)
* Members and above
* Full members and above 
* ...
* Nobody (most restrictive)